### PR TITLE
chore: revert "fix: update @libp2p/tcp (#6676)"

### PR DESCRIPTION
**Pull request by nflaig**

### CODE REVIEW

File: packages/light-client/test/mocks/LightclientServerApiMock.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import {concat} from "uint8arrays/concat";
```
Comment (jeluard) [inline packages/light-client/test/mocks/LightclientServerApiMock.ts:1]: Is this change related to the version downgrade? As `uint8arrays` is listed as a `devDependencies` of `light-client`, I wouldn't expect a change here?

Comment (nflaig) [reply packages/light-client/test/mocks/LightclientServerApiMock.ts:1]: I just reverted the commit

Comment (nflaig) [reply packages/light-client/test/mocks/LightclientServerApiMock.ts:1]: apparently related to the deps update, see https://github.com/ChainSafe/lodestar/pull/6676/commits/f3634b8789dc828d5101785ee389ac4e81b473f7

**Review by jeluard — APPROVED** [review]

**Review by wemeetagain — APPROVED** [review]

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.18.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.18.0) :tada:
